tests/qos/test_tunnel_qos_remap.py

```diff
@@ -161,7 +131,7 @@ def test_bounced_back_traffic_in_expected_queue(ptfhost, upper_tor_host, lower_t
     PKT_NUM = 100
 
     for dscp, queue in TEST_DATA:
-        pkt, _ = _build_testing_packet(src_ip=DUMMY_IP,
+        pkt, _ = build_testing_packet(src_ip=DUMMY_IP,
```

Comment:
`build_testing_packet` also returns the encapsulated packet (the second return value). But I didn't use the encapsulated packet because I only check the queue counter in this case. The bounced back packets have been checked in test case `test_encap_dscp_rewrite` (also in this module).